Make the HTML options the default

CWSpear opened the pull request:
This is to support file extensions that use the same syntax, but have a different extension, such as `.php`, `.twig`, `.hbs`, etc.

Resolves #140

CWSpear commented on wiredep.js:
```diff
@@ -85,7 +85,8 @@ function mergeFileTypesWithDefaults(optsFileTypes) {
   $._(optsFileTypes).each(function (fileTypeConfig, fileType) {
-    fileTypes[fileType] = fileTypes[fileType] || {};
+    // fallback to the default type for all html-like extensions (php, twig, hbs, etc)
+    fileTypes[fileType] = fileTypes[fileType] || fileTypes['default'];
```
These are the only lines of code that changed. Everything else is just tests.

stephenplusplus replied:
Can we just make this change and skip the extra tests? Also, somehow the permissions of this file changed from 644 -> 755. Can you bring that back?

shinnn commented:
LGTM

CWSpear commented:
(didn't mean to close)

Fixed permissions, removed tests.

Fun fact, the test file is called `wiredup_test.js`. Notice the `u` that should be an `e`. lol, it's always funny how long those kinds of typos can go on for...

stephenplusplus commented:
I'm not sure how I haven't merged this yet. I'm sorry everyone :(

@CWSpear you are officially granted all powers to merge solo without waiting for me.

> Fun fact, the test file is called wiredup_test.js. Notice the u that should be an e. It's always funny how long those kinds of typos can go on for...

Even funner fact, `wiredep` was once called `wiredup`, and it's only because of my confusion that we stuck with "ep" instead of "up" when I pushed to npm.

CWSpear commented:
Muahahahaha! This is the day the `wiredep` project starts its decent into trash-dom.

stephenplusplus commented:
Haha, I can't wait!

brunowego commented:
This deserves new tag :+1:

stephenplusplus commented:
You're definitely right about that. I need to make another fix, then we're heading straight towards 3.0 city. Buckle up!
